app/views/curation_concerns/base/_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr class="<%= dom_class(member) %> attributes">
+  <td class="thumbnail">
+    <% if member.reader? %>
```

Comment:
This line is causing test failures because it's calling `#reader?` on the `WorkShowPresenter`, and that method is only defined on the `FileSetPresenter`. Perhaps it needs to be defined on both classes.